src/blockchain/Blockchain.ts

```diff
@@ -128,12 +123,7 @@ export default class Blockchain implements IBlockchainApi {
     })
   }
 
-  public async getNonce(accountAddress: string): Promise<Codec> {
-    const nonce = await this.api.query.system.accountNonce(accountAddress)
-    if (!nonce) {
-      throw Error(`Nonce not found for account ${accountAddress}`)
-    }
-
-    return nonce
+  public async getNonce(accountAddress: string): Promise<u64> {
+    return this.api.query.system.accountNonce<u64>(accountAddress)
```

Comment:
What happens now when there is no nonce or can this not occur anymore?

Reply:
Can't happen - it will return the fallback value which is `0`.

Reply:
but this was always the case. The code was just misleading so far